src/components/SearchBar/styles.js

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+import * as colors from 'kitsu/constants/colors';
+
+export const styles = StyleSheet.create({
+  searchContainer: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: colors.white,
+    borderRadius: 3,
+    borderWidth: 0,
+  },
+  searchIcon: {
+    position: 'absolute',
+    backgroundColor: 'transparent',
+    padding: 10,
+  },
+  searchIconFocus: {
+    left: 0,
+    paddingRight: 0,
+  },
+  input: {
+    flex: 1,
+    height: 30,
+    textAlign: 'center',
+    paddingTop: 10,
+    paddingRight: 10,
+    paddingBottom: 10,
```

Comment:
I'd do:
```
paddingVertical: 10,
paddingRight: 10,
```

And I think 0 is the default padding value so it doesn't need to be declared for it to be set to 0.